app/server.rb

```diff
@@ -0,0 +1,71 @@
+require 'data_mapper'
+require 'rack-flash'
+require 'sinatra'
+require 'sinatra/partial'
+
+require './app/lib/tweet'
+require './app/lib/user'
+require_relative 'helpers/application'
+require_relative 'data_mapper_setup'
+
+env = ENV['RACK_ENV'] || 'development'
```

Comment:
Setting up env it's already done in data_mapper_setup so we are repeating ourselves here.